src/v3/src/util/formUtils.ts

```diff
@@ -77,8 +76,7 @@ export const getCustomButtonElements = (widgetProps: WidgetProps): ButtonElement
       options: {
         type: ButtonType.BUTTON,
         step: '',
-        dataSe: customButton.dataAttr,
-        classes: classNames(customButton.className, 'default-custom-button'),
```

Comment:
Can we now remove `classes` from the `ButtonElement` schema here? 
https://github.com/okta/okta-signin-widget/blob/a692da7a3705cd04e3af62bf469db193e3f0c07c/src/v3/src/types/schema.ts#L313